client/trino-cli/src/main/java/io/trino/cli/QueryPreprocessor.java

```diff
@@ -175,6 +176,7 @@ private static String preprocessQueryInternal(Optional<String> catalog, Optional
                     message = "\n===\n" + message + "\n===";
                 }
                 catch (Exception ignored) {
+                    // ignore
```

Comment:
All these "// ignore" comments are noisy and unnecessary. It is clear from the exception name and the empty body that the exception is being ignored.

Reply:
I wanted `{}` - but checkStyle complains.
Will do just `{
}`